packages/readiness/bundle/config/values.yaml

```diff
@@ -0,0 +1,7 @@
+#@data/values
+---
+namespace: default
+deployment:
+  hostNetwork: false
+  nodeSelector: null
+  tolerations: []
```

Comment:
Are the deployment related parameters relevant to us now?

Reply:
These are optional parameters, consumers of the package can configure if required.